mne/io/proj.py

```diff
@@ -12,6 +12,7 @@
 import numpy as np
 
 from .constants import FIFF
+from .fixes import _safe_svd
```

Comment:
```suggestion
from ..fixes import _safe_svd
```

(bad relative import)

```
Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/mne/bin/mne", line 5, in <module>
    from mne.commands.utils import main
  File "/home/runner/work/mne-python/mne-python/mne/__init__.py", line 44, in <module>
    from .io.pick import (
  File "/home/runner/work/mne-python/mne-python/mne/io/__init__.py", line 9, in <module>
    from .meas_info import (
  File "/home/runner/work/mne-python/mne-python/mne/io/meas_info.py", line 40, in <module>
    from .proj import (
  File "/home/runner/work/mne-python/mne-python/mne/io/proj.py", line 15, in <module>
    from .fixes import _safe_svd
ModuleNotFoundError: No module named 'mne.io.fixes'
```